### PR TITLE
Refracted findViewById to use ViewBinding 

### PR DESCRIPTION
**Pull request by dev-divyansh**
## Related Issues
App: #3796

Refracted findViewById to use ViewBinding  in [this Activity](https://github.com/owncloud/android/blob/000a0d97ac8b8f72f29f80818d64e17d0929e2fd/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt)

Please  review my PR and let me know what I  can  improve

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt [hunk 1]
```diff
@@ -48,19 +49,22 @@ open class FolderPickerActivity : FileActivity(),
...
     private lateinit var pickerMode: PickerMode
 
+    private  lateinit var binding : FilesFolderPickerBinding
     override fun onCreate(savedInstanceState: Bundle?) {
         Timber.d("onCreate() start")
 
         super.onCreate(savedInstanceState)
 
+        binding = FilesFolderPickerBinding.inflate(layoutInflater)
         setContentView(R.layout.files_folder_picker)
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:63]: Now that we introduced ViewBinding, we can set its root to the content view

```suggestion
        setContentView(binding.root)
```

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt [hunk 2]
```diff
@@ -48,19 +49,22 @@ open class FolderPickerActivity : FileActivity(),
...
 
         // Allow or disallow touches with other visible windows
-        val filesFolderPickerLayout = findViewById<LinearLayout>(R.id.filesFolderPickerLayout)
+
+        val filesFolderPickerLayout = binding.filesFolderPickerLayout
         filesFolderPickerLayout.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
```
Comment (abelgardep) [inline owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:68]: This variable is not used anywhere else, so I think we could remove it, what do you think?

```suggestion
        binding.filesFolderPickerLayout.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
```

**Review by abelgardep — CHANGES_REQUESTED** [review]
Some little details here @dev-divyansh and we are ready to go 🚀 

PD: The ktLint is failing because there are some unused imports and some additional spaces, reformat the code and it should be fixed 👍 
```
/bitrise/src/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:30:1 Unused import
/bitrise/src/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:31:1 Unused import
/bitrise/src/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:32:1 Unused import
/bitrise/src/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:56:35 Unexpected spacing before ":"
```

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt (hunk 1 above)
Comment (dev-divyansh) [reply owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:63]: Updated , made the changes

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt (hunk 2 above)
Comment (dev-divyansh) [reply owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:68]: yes , we can  directly use without creating the variable ;;
Updated it

**Review by abelgardep — APPROVED** [review]
LGTM! Thanks a lot for the contribution!!

PD: KtLint still failing because it needs a reformat 👍 
`/bitrise/src/owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.kt:54:46 Missing spacing after ":"`

**Comment by dev-divyansh** [comment]
Thankyou @abelgardep for guiding me and merging my PR  , looking forward to contribute more to the project 🚀🚀

**Pull request merged**